queries/groovy/highlights.scm

```diff
@@ -1,70 +1,261 @@
-(unit
-  (identifier) @variable)
+[
+  "!in"
+  "!instanceof"
+  "as"
+  "assert"
+  "case"
+  "catch"
+  "class"
+  "def"
+  "default"
+  "else"
+  "extends"
+  "finally"
+  "for"
+  "if"
+  "import"
+  "in"
+  "instanceof"
+  "package"
+  "pipeline"
+  "return"
+  "switch"
+  "try"
+  "while"
+  (break)
+  (continue)
+] @keyword
 
-(string
-  (identifier) @variable)
+[
+  "true"
+  "false"
+] @boolean
 
-(escape_sequence) @string.escape
+(null) @constant
 
-(block
-  (unit
-    (identifier) @module))
+"this" @variable.builtin
 
-(func
-  (identifier) @function)
+[
+  "int"
+  "char"
+  "short"
+  "long"
+  "boolean"
+  "float"
+  "double"
+  "void"
+] @type.builtin
 
-(number) @number
+[
+  "final"
+  "private"
+  "protected"
+  "public"
+  "static"
+  "synchronized"
+] @type.qualifier
 
-((identifier) @boolean
-  (#any-of? @boolean "true" "false" "True" "False"))
+(comment) @comment
```

Comment:
Are there specs for documentation comments? If yes, then having a capture for `@comment.documentation` will be nice

Reply:
Ah nice ! :tada: I didn't see `groovy_doc` below